src/markdown-pages/docs/managers-old.md

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
Please don't leave 'old' files around. If it's no longer relevant to the website, please remove it and we can always find it in the history of the repository if required.

Reply:
Thanks, this file has been removed now